src/main/docker/health-check.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Shouldn't we maybe have an endpoint in EI and EI-frontend for /health or something which just sends a response if it's alive or not instead? Will make it a lot easier to make this check then(only a short curl).

Reply:
We have same health check in all components, including components that don't have Java or web-server , why do different in one or few components?
Have same health-check and same configuration for all containers/images simplify it a lot, especially when same solution works in both Kubernetes and Docker, and other containers/images. This health-check checks so rabbitmq and/or mongodb is up and running before starting EI. EI backend stops executing if there is no RabbitMq messagebus and/or MongoDb.

Reply:
Maybe we should have had a discussion about how to handle this before implementing them in the other components as well? In my opinion, this is an unnecessary complex solution to make healthchecks. 

I think a healthcheck endpoint is more of a standard when reading different patterns for service based architectures and that works great different platforms as well(kubernetes, docker).

But since it's already implemented this way in the other services now, maybe it is the way to go.

Reply:
This is no implementation in the Eiffel cmpoment, only in image/container for handling the MessageBus/MongDb issues, so Eiffel components is not started before MessageBus and MongoDb is up and running.

If you want to implement a health check, take it up with the Eiffel Architects and if approved, implement it in all Eiffel and non-Eiffel components.
The current solution is already used today in Kubernetes and Docker in Eiffel Sandbox and similar environments.

When we have some built in health check inside Eiffel components, then we can switch to use them. But today we don't have it.
And this check is not checking if EI is up or not, It check if MessageBus and MongoDb is up and running.